### PR TITLE
Added tests for IAM policies for bucket operations

### PR DESCRIPTION
**Pull request by shtripat**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Motivation and Context
Make sure all IAM policies come in effect and work as expected.

## How to test this PR?
`make test-pbac`

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

File: docs/bucket/policies/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Bucket Access Management Quickstart Guide [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/)
+
+MinIO uses Policy-Based Access Control (PBAC) to define the authorized actions and resources to which an authenticated user has access. Each policy describes one or more `actions` and `conditions` that outline the permissions of a user or group of users.
+
```
Comment (harshavardhana) [inline docs/bucket/policies/README.md]: this is not a bucket policy instead its an IAM policy directly attached to the user

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: docs/bucket/policies/README.md (hunk 1 above)
Comment (shtripat) [reply docs/bucket/policies/README.md]: ah ok. better rename the tests as `docs/iam/policies`. will do the needed changes.

Comment (harshavardhana) [reply docs/bucket/policies/README.md]: this documentation is not needed @shtripat

Comment (shtripat) [reply docs/bucket/policies/README.md]: > this documentation is not needed @shtripat

ok, will remove this. thanks!

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**